internal/renderers/skia/lib.rs

```diff
@@ -52,72 +52,63 @@ cfg_if::cfg_if! {
 
 /// Use the SkiaRenderer when implementing a custom Slint platform where you deliver events to
 /// Slint and want the scene to be rendered using Skia as underlying graphics library.
-pub struct SkiaRenderer<NativeWindowWrapper> {
+pub struct SkiaRenderer {
     window_adapter_weak: Weak<dyn WindowAdapter>,
     rendering_notifier: RefCell<Option<Box<dyn RenderingNotifier>>>,
-    canvas: RefCell<Option<SkiaCanvas<DefaultSurface, NativeWindowWrapper>>>,
+    image_cache: ItemCache<Option<skia_safe::Image>>,
+    path_cache: ItemCache<Option<(Vector2D<f32, PhysicalPx>, skia_safe::Path)>>,
+    rendering_metrics_collector: RefCell<Option<Rc<RenderingMetricsCollector>>>,
+    surface: DefaultSurface,
 }
 
-impl<
-        NativeWindowWrapper: raw_window_handle::HasRawWindowHandle + raw_window_handle::HasRawDisplayHandle,
-    > SkiaRenderer<NativeWindowWrapper>
-{
+impl SkiaRenderer {
     /// Creates a new renderer is associated with the provided window adapter.
-    pub fn new(window_adapter_weak: Weak<dyn WindowAdapter>) -> Self {
-        Self {
+    pub fn new(
+        window_adapter_weak: Weak<dyn WindowAdapter>,
+        native_window: &impl raw_window_handle::HasRawWindowHandle,
+        native_display: &impl raw_window_handle::HasRawDisplayHandle,
```

Comment:
Not for this PR, but i think eventually we want to just take a WindowHandle and DisplayHandle struct.  unless we need to take ownership of it, but that's not the case it seems.

Reply:
Agreed.